### ISSUE TITLE
[TW-1195] How To: task best practices

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1230]
_Paul Beckingham on 2009-05-28T11:05:27Z says:_ 

 I would like to see a best practices document, that shows several "tips" on how to use task in a way that doesn't end up being counter-productive.

It's one thing to set up task correctly and start using it, but it's another thing entirely to use it well.  This site is going to be filled with documents describing how to use certain esoteric features, but that will be overwhelming.  SImilar to the "30 second tutorial" page on the old site, we need some distilled wisdom.

This might include a discussion on use of nested projects, appropriate use of tags, what does and does not warrant a sensible annotation, etc.  Perhaps it would include comments on certain .taskrc tweaks (color _this_ kind of task, set _this_ as a default command, etc), sensible use of DropBox, starting each task description with a verb, etc.

This might include a discussion on how to avoid being overwhelmed, by creating fewer, more general tasks, rather than more finer-grained tasks.

Basically it should be a sensible, practical guide to getting task to _help_ you.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2009-05-28T11:05:27Z
Modified: 2014-02-09T02:39:08Z
```
